serializer.rst

```diff
@@ -138,11 +138,53 @@ To use annotations, first add support for them via the SensioFrameworkExtraBundl
     $ composer require sensio/framework-extra-bundle
 
 Next, add the :ref:`@Groups annotations <component-serializer-attributes-groups-annotations>`
-to your class and choose which groups to use when serializing::
+to your class::
+
+    // src/Entity/Product.php
+    namespace App\Entity;
+    
+    use Doctrine\ORM\Mapping as ORM;
+    use Symfony\Component\Serializer\Annotation\Groups;
+    
+    /**
+    * @ORM\Entity(repositoryClass="App\Repository\ProductRepository")
```

Comment:
```suggestion
    * @ORM\Entity()
```

repository seems out of context here, wdyt?

Reply:
Yes you are right